packages/@aws-cdk/aws-cognito/test/user-pool.test.ts

```diff
@@ -331,6 +331,65 @@ describe('User Pool', () => {
     });
   });
 
```

Comment:
Can you add a new integration test of this nature - https://github.com/aws/aws-cdk/blob/c62377e14caae677deb7e4eae692eaccb2020c67/packages/%40aws-cdk/aws-cognito/test/integ.user-pool-idp.apple.ts#L4-L12 - with verification steps on how you verified that this actually worked?

(Just for one of them - either email or sms - is sufficient)